### PR TITLE
terraform-0.14 package added

### PR DESCRIPTION
**Pull request by maximmi**
## what
* terraform 0.14-rc1 added

## why
* to be able to start using and testing modules with 0.14 version

### CODE REVIEW

File: vendor/terraform-0.14/Makefile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+export PACKAGE_NAME = $(MASTER_PACKAGE_NAME)-$(MAJOR_VERSION)
+
+include ../../tasks/Makefile.package
+include ../../tasks/Makefile.apk
+include ../../tasks/Makefile.deb
+
+# NOTE: This build uses a custom build template ./APKBUILD
+export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
+export DOWNLOAD_URL ?= https://releases.hashicorp.com/terraform/$(PACKAGE_VERSION)/terraform_$(PACKAGE_VERSION)_$(OS)_$(ARCH).zip
+export AUTO_UPDATE_ENABLED = false
```
Comment (Nuru) [inline vendor/terraform-0.14/Makefile:16]:
```suggestion
```

File: vendor/terraform-0.14/Makefile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Package details
+export VENDOR ?= hashicorp
+export PACKAGE_REPO_NAME = terraform
+export PACKAGE_EXE = terraform
+export MASTER_PACKAGE_NAME = terraform
+export MAJOR_VERSION = 0.14
```
Comment (Nuru) [inline vendor/terraform-0.14/Makefile:6]:
```suggestion
export MAJOR_VERSION = 0.14
export PACKAGE_PRERELEASE_ENABLED = true
```

File: vendor/terraform-0.14/terraform_0.14.post-deinstall [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# This is an Alpine `deinstall` hook that removes the alternative
+MASTER_PACKAGE_NAME=terraform
+MAJOR_VERSION=0.14
+PACKAGE_NAME=${MASTER_PACKAGE_NAME}-${MAJOR_VERSION}
+INSTALL_DIR=/usr/share/${MASTER_PACKAGE_NAME}/${MAJOR_VERSION}/bin
+
+update-alternatives --remove ${PACKAGE_NAME} ${INSTALL_DIR}/${PACKAGE_NAME} --quiet
+update-alternatives --remove ${PACKAGE_NAME}-direnv ${INSTALL_DIR}/${PACKAGE_NAME} --quiet
+update-alternatives --remove ${MASTER_PACKAGE_NAME} ${INSTALL_DIR}/${PACKAGE_NAME} --quiet
```
Comment (Nuru) [inline vendor/terraform-0.14/terraform_0.14.post-deinstall:10]:
```suggestion
update-alternatives --remove ${PACKAGE_NAME} ${INSTALL_DIR}/${MASTER_PACKAGE_NAME} --quiet
update-alternatives --remove ${PACKAGE_NAME}-direnv ${INSTALL_DIR}/${MASTER_PACKAGE_NAME} --quiet
update-alternatives --remove ${MASTER_PACKAGE_NAME} ${INSTALL_DIR}/${MASTER_PACKAGE_NAME} --quiet
```

**Review by Nuru — CHANGES_REQUESTED** [review]
Do not create a `terraform_0.14` package. The packages with underscores are obsolete, we maintain the existing ones for people who use them, but we encourage everyone to switch to the packages with dashes.

File: vendor/terraform-0.14/Makefile (hunk 2 above)
Comment (maximmi) [reply vendor/terraform-0.14/Makefile:6]: done

**Review by osterman — DISMISSED** [review]

**Review by Nuru — DISMISSED** [review]

**Review by Nuru — APPROVED** [review]

**Pull request merged**